srcalc/src/main/java/gov/va/med/srcalc/web/view/admin/EditRiskModel.java

```diff
@@ -17,6 +33,146 @@
+            termType = type;
+        }
+         
+        public String getDisplayName()  
+        {
+            return displayName;
+        }
+        
+        public String getTermType() 
+        {
+            return termType;
+        }
+        
+        public float getCoefficient()
+        {
+            return targetTerm.getCoefficient();
+        }
+
+        public static ModelTermSummary createTermSummary( ModelTerm modelTerm ) 
+        {
+            String dispName="";
+            String type="";
+            
+            if( modelTerm instanceof ConstantTerm ) 
+            {
+                dispName = "Constant"; 
+                type = "Constant"; // or N/A or blank
+            }
+            else if( modelTerm instanceof ProcedureTerm )
+            {
+                type = "Multiplier"; // ???
+                dispName = "Procedure (RVU Multiplier)"; 
+            }
+            else if( modelTerm instanceof BooleanTerm )
+            {
+                type = "Boolean";
+                BooleanTerm boolTerm = (BooleanTerm)modelTerm;                
+                dispName = boolTerm.getVariable().getDisplayName();
+            }            
+            else if( modelTerm instanceof NumericalTerm )
+            {
+                type = "Numerical";
+                NumericalTerm numTerm = (NumericalTerm)modelTerm;
+                dispName = numTerm.getVariable().getDisplayName();                                     
+            }
+            else if( modelTerm instanceof DiscreteTerm )
+            {
+                DiscreteTerm discreteTerm = (DiscreteTerm)modelTerm;                                
+                MultiSelectOption opt = discreteTerm.getOption();
+                DiscreteVariable var = discreteTerm.getVariable();
+                
+                if( var instanceof DiscreteNumericalVariable ) 
+                {
+                    type = "Discrete Numeric";
+                    dispName = var.getDisplayName()+ " = " + opt.getValue();
+                }
+                else if( var instanceof MultiSelectVariable ) 
+                {
+                    type = "Discrete";
+
+                    MultiSelectVariable msv = (MultiSelectVariable)var;
+                    msv.getOptions();
+                    dispName = var.getDisplayName()+ " = " + opt.getValue() ;
+                }
+            }            
+            else if( modelTerm instanceof DerivedTerm )
+            {
+                type = "Rule";
+                dispName = "Rule: "+((DerivedTerm)modelTerm).getRule().getId();
+            }
+            else
+            {
+                // or throw an error?
+                dispName = "Unrecognized Term: "+modelTerm.getClass().getName();    
+                type = "";
+            }
+            
+            // Use new Type column to display the Type.
+            // Can still do this if that's what they want.
+            // dispName = dispName + " ("+type+")";
```

Comment:
Kind of. This Type column was a deviation from the mockup which hasn't been approved and so this was just kind of and indication that it wasn't finalized and could be changed. But I suppose that is known and a comment doesn't really do much.   We could employ 'TODO' (or similar) markers which eclipse can recognized and list in a View. I think its called a Marker view or something?